backend/installation/README.md

```diff
@@ -1,55 +1,78 @@
 # Backend Installation
 
-{% hint style="warning" %}
-This documentation should be split into a **local** and a **docker** installation variant. Also maybe there should be a main docker installation guide for the whole system at once!?
-{% endhint %}
 
 ### Clone Repository
 
 The Backend Repository can be found on github.  
 [https://github.com/Human-Connection/Nitro-Backend](https://github.com/Human-Connection/Nitro-Backend)
 
-## Fork the repo
+### Fork the repo
 
 Click on the fork button.
 
-![Fork screenshot](https://i.imgur.com/syoN9rF.png)
+![Fork screenshot](../../.gitbook/assets/screenshot-forking-nitro-backend.png)
 
-After you have forked, modify the following command to add your Github user name.
+After you have forked and before you clone, create a folder on your computer to where the Nitro-Backend folders and files of your fork will be cloned (copied). In your terminal or commandline tool make this folder the actual path.
```

Comment:
You have overlooked it, I think.

Ah okay. I am very new on GitHub as well as new to the terminal commands and still learning.
I remember now, that I have just set the current working directory to the one where I like to have the Nitro-Backend copied into.

Will you correct the text as you like or will you just use the old version?

Reply:
No, I would suggest you to change the text and maybe add a hint how to bring changes in a fork back to the origin repository.

Reply:
Actually this is a good training for you now. Apparently you are editing your own fork of the documentation: `Tirokk:Tirokk-Docker-Neo4j-1-1` You could try to work in a local repository on your laptop and push/pull commits to/from two different remotes: `origin` and `Tirokk`

If that's a little confusing you can always ask @mattwr18 or @FedericoEsparza for help :laughing:

Reply:
Or we can merge right away if it's too complicated and update the section later.

Reply:
It’s a good training @roschaefer , clearly! ☺️😁
I’ll make the change myself in GitHub-Web in my branch ´Tirokk:Tirokk-Docker-Neo4j-1-1.
Will you see the change than automatically or do I have to make a new PR?

> … and maybe add a hint how to bring changes in a fork back to the origin repository.

I would suggest to make an Issue out of this, because that’s kind of a short documentation where we show and how we recommend to use GitHub, in my mind. And it belongs to all repos of HC, not only to the Nitro-Backend. 🤔
Okay for you?

Reply:
Github has also feature to check out a forked branch **without** adding a remote. But it's a pity that it is [read only](https://help.github.com/articles/checking-out-pull-requests-locally/#error-failed-to-push-some-refs). So we cannot use it for our documentation.

@Tirokk maybe you research a good tutorial for Github's fork/pull-request workflow and simply put in a reference?

Reply:
Is the following link a good reference, @roschaefer ?
[GitHub Standard Fork & Pull Request Workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
Or do we have to change or clarify anything about it?

If it's acceptable for you:
– I would put it in the documentation right now!
– Should we than setup a little discussion in discord about it? And at our next Community Meeting next week?
– Or just put it as a reference in the documentation and see what happens?

I would recommend to create a general GitHub section in our documentation and reference to that from all our installation instructions.
Here we than would have the possibility to clarify and correct little things for our HC workflow.

PS: The guy from the Chaser324 project made a very good advertisement for his project as he invested time to write this detailed and good description. He got 1536 stars and 425 forks.
May we could fork it by our self if we need to clarify or make some corrections …

Reply:
I have committed  my change directly in `Tirokk:Tirokk-Docker-Neo4j-1-1` @roschaefer .
Though I am not sure you get a message that I have done so, I write you here.